### DISCUSSION TITLE
New Feature: support for separate label style if active

### DISCUSSION THREAD

**Discussion by dlindstrm** [callstack/react-native-bottom-tabs#213] [Ideas]
### Why it is needed?

It would be nice to be able to have a different label style depending on active/inactive state on Android. For example black and bold label if active vs grey and regular font weight if inactive, like Google Maps:
![image](https://github.com/user-attachments/assets/b4e8388f-6d22-4738-ac9b-2a4e5d78fdee)

**Comment by okwasniewski** [comment]
Hey, thanks for opening feature request. I'm moving this to discussions because all feature requests will now be under the "Ideas 💡" tab in GitHub discussions.
